cmd/gentypes/gentypes.go

```diff
@@ -420,6 +429,24 @@ package dap
 type Message interface {
 	GetSeq() int
 }
+// RequestMessage is an interface implemented by all Request-types. 
```

Comment:
Please leave a blank line between types here too, just for clarity (gofmt handles this for the generated code)